### DISCUSSION TITLE
Add toggle to disable automatic scrolling to object after placement

### DISCUSSION THREAD

**Discussion by 641i130** [ppy/osu#17424] [Ideas] [answered]
The option to turn off the auto-scroll when writing a beatmap in mania would be very useful.

**Comment by bdach** [comment]
to clarify: are you asking that placing a note should not seek the track to where the note was placed?

this should probably be available/set for all scrolling rulesets if so.

**Reply by 641i130** [reply]
Correct, I agree!

I think another useful thing that could complement this setting would be the playback to position option. 

Like pressing the spacebar (while in compose mode) it plays, then when you press the spacebar again it would jump back to where it started. You could play without this setting by pressing shift space (to prevent it from jumping back to where it started I think). This would greatly improve the composition system!!!

**Reply by peppy** [reply]
Note that this isn't just scrolling rulesets - the same behaviour applies to distance snap mode in osu! for instance. Making this a toggle is fine, it's going to be a per-user preference that while feels like a good default, is not how stable worked.

**Comment by bdach** [comment, marked as answer]
Made https://github.com/ppy/osu/issues/22668 out of this since it feels that the direction is more or less established (and it comes up relatively frequently).
